bin/sdk_generator.sh

```diff
@@ -25,40 +25,34 @@
 
 set -e
 
-GLEAN_PARSER_VERSION=4.4.0
+GLEAN_PARSER_VERSION=6.1.1
```

Comment:
This should be 6.1.2.
Did you copy it from https://github.com/mozilla/glean/blob/v51.1.0/glean-core/ios/sdk_generator.sh ?

Reply:
Ooops I think I copied the one in iOS which was for glean 51.0.1 and not 51.1.0 🤦 Updated!